framework/Runners/DistributedMemoryRunner.py

```diff
@@ -40,27 +41,20 @@ class DistributedMemoryRunner(InternalRunner):
     Class for running internal objects in distributed memory fashion using
     ppserver
   """
-  def __init__(self, messageHandler, ppserver, args, functionToRun,
+  def __init__(self, messageHandler, args, functionToRun,
                      frameworkModules = [], identifier=None, metadata=None,
-                     functionToSkip = None, uniqueHandler = "any",
-                     profile = False):
+                     uniqueHandler = "any", profile = False):
     """
       Init method
       @ In, messageHandler, MessageHandler object, the global RAVEN message
         handler object
-      @ In, ppserver, ppserver, instance of the ppserver object
       @ In, args, dict, this is a list of arguments that will be passed as
         function parameters into whatever method is stored in functionToRun.
         e.g., functionToRun(*args)
       @ In, functionToRun, method or function, function that needs to be run
-      @ In, frameworkModules, list, optional, list of modules that need to be
-        imported for internal parallelization (parallel python). This list
-        should be generated with the method returnImportModuleString in utils.py
```

Comment:
frameworkModules is still listed in init input but the docstring is removed. Please check and update

Reply:
removed